### ISSUE TITLE
Global variables should be declared with `const` in Julia.

### ISSUE THREAD

**Issue by bicycle1885** [colmnet/ragel#5]
In Julia, global variables should be declared with `const` if they are constant. This is because the Julia compiler fails to inter types of global variables without `const`s, which leads to large [performance degradation](http://julia.readthedocs.org/en/latest/manual/performance-tips/#avoid-global-variables).

For example, in our parsers, the following variables
```julia
_fastaparser_start  = 6
_fastaparser_first_final  = 6
```
should be declared as
```julia
const _fastaparser_start  = 6
const _fastaparser_first_final  = 6
```

This already has been fixed in our fork and the patch fixes just two lines: https://github.com/BioJulia/ragel7/commit/ba684f9ecb0bcb5c318d640ee7e5f34cdd8ce767. Could you include this patch before the final release?

---

```
@@ -351,12 +351,12 @@ namespace julia_gen
  		}
  		case [A: static_array] {
  			send Parser
 -				"[A.ident] = [type(A.type)]"
 +				"const [A.ident] = [type(A.type)]"
  					"\[ [num_list(A.num_list)] \]
  		}
  		case [V: static_value] {
  			send Parser
 -				"[V.ident] = [V.number]
 +				"const [V.ident] = [V.number]
  		}
  		# case [declaration]
  		case [
```

**Comment by RReverser** [comment]
@bicycle1885 Why don't you send PR? It's a better format than issue for fix suggestions.

**Comment by bicycle1885** [comment]
Oh, I thought this is a mirror repository. OK, I'll make a pull request.

**Comment by bicycle1885** [comment]
I made a pull request at #6 .

**Comment by RReverser** [comment]
Well, yeah, it's a mirror, but I believe it's two-sides and contributions in forms of PRs are easy to merge & push to origin (at least that's how I sent my changes too).

**Comment by bicycle1885** [comment]
Thanks a lot. I'll close this issue since my patch has been merged in #6 .
